index.html

```diff
@@ -5,34 +5,55 @@
   <meta charset="utf-8">
 
   <title>SpaceWalkers</title>
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
   <link href="css/index.css" rel="stylesheet">
-
+  <link rel="stylesheet/less" type="text/css" href="./less/index.less"/>
```

Comment:
nah. why would you import LESS into the html?
You already have your CSS, so you need to compile your LESS files into CSS.